compiler/vm.nim

```diff
@@ -931,9 +1017,14 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
         # we know these cannot be equal
         regs[ra].intVal = ord(false)
       else:
-        regs[ra].intVal = ord((regs[rb].node.kind == nkNilLit and
-                              regs[rc].node.kind == nkNilLit) or
-                              regs[rb].node == regs[rc].node)
+        let nb = regs[rb].node
+        let nc = regs[rc].node
+        template getTyp(n): untyped =
+          n.typ.skipTypes({tyAlias})
```

Comment:
Why not ``skipTypes(abstractInst)`` here?

Reply:
done